source/scss/_library/_base.main.scss

```diff
@@ -4,11 +4,15 @@
 
 body {
   background: $c-body-bgcolor;
-  font: 400 100%/1.3 $ff-font;
+  font: 400 100%/1.3 $ff-font--primary;
   -webkit-text-size-adjust: 100%;
   color: $c-body-color;
   overflow-x: hidden;
   -webkit-font-smoothing: antialiased;
   -moz-font-smoothing: antialiased;
```

Comment:
this should actually be `-moz-osx-font-smoothing: grayscale;` for antialiasing on firefox